python/foglamp/common/utils.py

```diff
@@ -37,6 +37,27 @@ def check_reserved(string):
     return True
 
 
+def check_foglamp_reserved(string):
+    reserved = [
+        'foglamp',
+        'general',
+        'advanced',
+        'notifications',
+        'north',
+        'south',
+        'filter',
+        'notify',
+        'rule',
+        'delivery',
```

Comment:
rule and delivery are also reserved?

Reply:
yes, these have a pre defined meaning in Foglamp, so should not be used as service name.